docs/rackhd/tftp_dhcp_server.rst

```diff
@@ -0,0 +1,313 @@
+| DHCP service    +--------------------------------------------------------------+------------------------------------------------------------+
+| only            | DHCP service could not proxy DHCP, and on-dhcp-proxy         | Replace "autoboot" command in `Default iPXE Config`_ with  |
+|                 | also could not be deployed in the DHCP server, only bootfile | "dhcp" and "http://172.31.128.1:9080/api/common/profiles", |
+|                 | name could be specified by DHCP                              | then re-compile iPXE in `on-imagebuilder`_ to generate new |
+|                 |                                                              | iPXE files, specify one of generated iPXE files as         |
+|                 |                                                              | as bootfile name in DHCP configuration. IP address and     |
+|                 |                                                              | port are configured according to RackHD southbound         |
+|                 |                                                              | configuration. Two drawbacks for this solution due to      |
+|                 |                                                              | DHCP and environment limitations:                          |
+|                 |                                                              | 1. IP address and port are hardcoded in iPXE file          |
+|                 |                                                              | 2. Only one iPXE bootfile name could be specified. it's    |
+|                 |                                                              | not flexiable to switch bootfile name automatically.       |
++-----------------+--------------------------------------------------------------+------------------------------------------------------------+
+| Third-party     | DHCP service's funtionality is less than ISC DHCP,           | **on-dhcp-proxy** could be leveraged to avoid complicated  |
+| DHCP service    | but it could proxy DHCP like ISC DHCP's configuration        | DHCP configuration.                                        |
+| + DHCP proxy    | "option vendor-class-identifier "PXEClient"                  |                                                            |
++-----------------+--------------------------------------------------------------+------------------------------------------------------------+
+
+
+TFTP and DHCP are in Separated Server
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+RackHD default TFTP and DHCP service such as on-tftp, on-dhcp-proxy and ISC DHCP could be deployed
+in another separated server after some simple configurations.
+
+RackHD also could disable its own TFTP and DHCP service, and leverage users' TFTP and DHCP server
+inpi/common/profiles datacenter or lab environments.
```

Comment:
I think what you mean here is "RackHD may be deployed without own TFTP and DHCP service, and leverage an existing TFTP and DHCP server with configuration to call back to `api/common/profiles` from datacenter or lab environments."